src/components/require-mfa.test.tsx

```diff
@@ -1,25 +1,57 @@
 import { TestingProviders } from '@/tests/providers'
 import { useUser } from '@clerk/nextjs'
-import { render, act } from '@testing-library/react'
+import { render, act, waitFor } from '@testing-library/react'
 import router from 'next-router-mock'
-import { beforeEach, expect, it, vi, type Mock } from 'vitest'
+import { beforeEach, describe, expect, it, vi, type Mock } from 'vitest'
 import { RequireMFA } from './require-mfa'
 
 vi.mock('@/server/actions/org.actions', () => ({
     getReviewerPublicKeyAction: vi.fn(() => Promise.resolve(null)),
 }))
 
-beforeEach(() => {
-    router.setCurrentUrl('/')
-})
+describe('RequireMFA', () => {
+    describe('when MFA is not enabled', () => {
+        beforeEach(() => {
+            router.setCurrentUrl('/')
+        })
+
+        it('redirects if mfa is not set', async () => {
+            ;(useUser as Mock).mockImplementation(() => ({ user: { twoFactorEnabled: false } }))
```

Comment:
another advantage of mockClerkSession is that it returns the mocks, so this could then call mockImplementation on them